docusaurus/docs/core-concepts/config-store/standard-config-type-host.v1.md

```diff
@@ -0,0 +1,45 @@
+# Standard Config Type host.v1
+
+A Config of Config Type `host.v1` is used by a Ziti tunneler application to forward packets to a destination server for a particular Ziti service.
+
+## Examples
+
+This Config forwards Ziti service traffic to a server known as `tcp:localhost:54321`. The server must be reachable at that destination from the network perspective of the device where the Ziti tunneler is running.
+
+```json
+{
+    "address": "localhost",
+    "port": 54321,
+    "protocol": "tcp"
+}
+```
+
+Rather than a static destination port, this Config forwards the Ziti service traffic to the same TCP port that was used by the initiating client as long as it falls within the range of allowed ports.
+
+```json
+{
+    "address": "localhost",
+    "forwardPort": true,
+    "allowedPortRanges": [ { "low": 8000, "high": 8888 } ],
+    "protocol": "tcp"
+}
+```
+
+Rather than a static destination address, this Config forwards the Ziti service traffic to the same address that was used by the initiating client as long as it is in the list of allowed addresses.
```

Comment:
> to the same address that was used by the initiating client

"used by" might be ambiguous, especially to the uninitiated? alternatives that sound better to me:

- to the same address that was intercepted by the intercepting tunneler
- to the same address that the initiating (intercepted?) client was connecting to